deploy/env/dev.yaml

```diff
@@ -11,4 +11,13 @@ registration-service:
                   "resource": "crt",
                   "clientId": "crt",
                   "public-client": true
-                }'
\ No newline at end of file
+                }'
+  verification.enabled: 'true'
+  verification.daily_limit: '3'
+  verification.attempts_allowed: '5'
```

Comment:
This should be in a different format: 
```yaml
  verification:
    enabled: 'true'
    daily-limit: '3'
    attempts-allowed: '5'
```
Btw, why do you use the underscore in the param names? This can be confusing since the underscore is the default delimiter in the var names

Reply:
this doesn't need to be here since they are already set as defaults in configuration.go